docs/services/vision/detection.md

```diff
@@ -232,13 +232,18 @@ The following code gets the robot’s vision service and then runs a color detec
 from viam.services.vision import VisionClient, VisModelConfig, VisModelType
 
 robot = await connect()
-# grab camera from the robot
+# Grab camera from the robot
 cam1 = Camera.from_robot(robot, "cam1")
-# grab Viam's vision service for the detector
+# Grab Viam's vision service for the detector
 my_detector = VisionClient.from_robot(robot, "my_detector")
 
+detections = await my_detector.get_detections_from_camera(cam1)
+
+# If you need to get an image first and then run detections on it,
+# you can do it this way (generally slower but useful if you need to,
+# for instance, crop the image before running the detector on it):
```

Comment:
```suggestion
# you can do it this way (generally slower but useful if you need to use the image
# afterwards):
```
^ cropping you could define a transform camera for - you only need to do this if you want to save the image after finding something on it for example . Like if it's a bird camera and you detected a bird you'd want to save the image. You can only do that if you still have the image otherwise you'd need to take another image and you don't know if the bird would still be there

Reply:
```suggestion
# you can do it this way (generally slower but useful if you need to
# use the image afterwards
```